Storage/SQL/BrowserDB.swift

```diff
@@ -326,7 +326,26 @@ extension BrowserDB {
     }
 
     func run(sql: String, withArgs args: Args? = nil) -> Success {
-        return self.write(sql, withArgs: args) >>> succeed
+        return run([(sql, args)])
+    }
+
+    func run(sql: [(sql: String, args: Args?)]) -> Success {
```

Comment:
We should probably document that this will run these all in one transaction and will block the thread; that's usually what we want, but could surprise.